Packs/Jira/Integrations/JiraV2/JiraV2.py

```diff
@@ -819,12 +1246,22 @@ def main():
         elif demisto.command() == 'jira-get-id-offset':
             get_id_offset()
 
+        elif demisto.command() == 'get-mapping-fields':
+            return_results(get_mapping_fields_command())
+
+        elif demisto.command() == 'update-remote-system':
+            return_results(update_remote_system_command(demisto.args()))
+
         elif demisto.command() == 'get-remote-data':
-            return_results(get_remote_data_command(**demisto.args()))
+            return_results(get_remote_data_command(demisto.args()))
 
         elif demisto.command() == 'jira-get-id-by-attribute':
             return_results(get_account_id_from_attribute(**demisto.args()))
 
+        elif demisto.command() == 'jira-list-transitions':
+            return_results(list_transitions_command(demisto.args()))
+        elif demisto.command() == 'get-modified-remote-data':
+            return_results(get_modified_remote_data_command(demisto.args()))
```

Comment:
see that inn some commands we have `**snakify(demisto.args()` and in yours no. we need to understand if it makes sense

Reply:
We've talked about it and said we can leave it this way.